source/frontend/src/features/disposition/list/DispositionFilter/DispositionFilter.tsx

```diff
@@ -102,7 +102,7 @@ export const DispositionFilter: React.FC<IDispositionFilterProps> = ({
                     placeholder="Team Member"
                   />
                 </Col>
-                <Col xl="4">
+                <Col xl="5">
```

Comment:
Looks like this was a hack to work around horizontal alignment issues from the selectInput above this component, no longer necessary.